src/cpp/dynamic-types/TypeDescriptor.cpp

```diff
@@ -252,10 +252,10 @@ bool TypeDescriptor::is_type_name_consistent(
     // The first letter must start with a letter ( uppercase or lowercase )
     if (sName.length() > 0 && std::isalpha(sName[0]))
```

Comment:
``` suggestion
    if (!sName.empty() > 0 && std::isalpha(sName[0]))
```

Reply:
std::isalpha takes current C locale into account, that doesn't seem correct at this point?